tests/functional/csp-simple.js

```diff
@@ -0,0 +1,13 @@
+require.config({
```

Comment:
Any reason to not write these as TS? Not that it really matters with them being this small...